.goreleaser.yml

```diff
@@ -14,7 +14,11 @@ before:
     # run `go mod tidy`. The `/bin/sh -c '...'` is because goreleaser can't find cd in PATH without shell invocation.
     - /bin/sh -c 'cd ./caddy-build && go mod tidy'
     - git clone --depth 1 https://github.com/caddyserver/dist caddy-dist
+    - mkdir -p caddy-dist/man
```

Comment:
I believe `manpage` creates the directory if it does not exist, right?

Reply:
Yep, it slipped my mind, but the code confirms it.